s3_storage_client.go

```diff
@@ -80,11 +80,7 @@ func (util *snowflakeS3Client) getFileHeader(meta *fileMetadata, filename string
 		if errors.As(err, &ae) {
 			if ae.ErrorCode() == notFound {
 				meta.resStatus = notFoundFile
-				return &fileHeader{
-					digest:             "",
-					contentLength:      0,
-					encryptionMetadata: nil,
-				}, nil
+				return nil, errors.New("could not find file")
```

Comment:
why `errors.New` not  `fmt.Errorf` like in other cases ?

Reply:
I've changed it back to fmt.Errorf to match with other cases